CHANGELOG.md

```diff
@@ -43,6 +43,11 @@ All notable changes to this project will be documented in this file.
     * #### Added
     * #### Changed
     * #### Removed
+* ### NI Switch Executive
```

Comment:
Add this to the template at the bottom of the file.

Reply:
done